pkgs/tools/misc/blahaj/default.nix

```diff
@@ -1,6 +1,9 @@
 { lib
+, stdenv
 , crystal
 , fetchFromGitHub
+, mt ? true
```

Comment:
"mt" is not very expressive. First of all, what does 'mt' mean? Ok, multithreading. But multi-threading at build-time or run-time? Does this respect NIX_BUILD_CORES?

Reply:
This is run time multithreading as per https://crystal-lang.org/2019/09/06/parallelism-in-crystal/